ballerina/Package.md

```diff
@@ -1,17 +1,158 @@
+
+Before proceeding with the Quickstart, ensure you have obtained the Access Token using the following steps:
+
+1. Create an authorization URL using the following format:
+
+   ```
+   https://app.hubspot.com/oauth/authorize?client_id=<YOUR_CLIENT_ID>&scope=<YOUR_SCOPES>&redirect_uri=<YOUR_REDIRECT_URI>
+   ```
+
+   Replace the `<YOUR_CLIENT_ID>`, `<YOUR_REDIRECT_URI>` and `<YOUR_SCOPES>` with your specific value.
+
+2. Paste it in the browser and select your developer test account to intall the app when prompted.
+    
+   <img src="../docs/resources/install_app.png" alt="Hubspot app creation 1 testacc3" style="width: 70%;">
+
+3. A code will be displayed in the browser. Copy the code.
+
+   ```
+   Received code: na1-129d-860c-xxxx-xxxx-xxxxxxxxxxxx
+   ```
+
+4. Run the following curl command. Replace the `<YOUR_CLIENT_ID>`, `<YOUR_REDIRECT_URI`> and `<YOUR_CLIENT_SECRET>` with your specific value. Use the code you received in the above step 3 as the `<CODE>`.
+
+   - Linux/macOS
+
+     ```bash
+     curl --request POST \
+     --url https://api.hubapi.com/oauth/v1/token \
+     --header 'content-type: application/x-www-form-urlencoded' \
+     --data 'grant_type=authorization_code&code=<CODE>&redirect_uri=<YOUR_REDIRECT_URI>&client_id=<YOUR_CLIENT_ID>&client_secret=<YOUR_CLIENT_SECRET>'
+     ```
+
+   - Windows
+
+     ```bash
+     curl --request POST ^
+     --url https://api.hubapi.com/oauth/v1/token ^
+     --header 'content-type: application/x-www-form-urlencoded' ^
+     --data 'grant_type=authorization_code&code=<CODE>&redirect_uri=<YOUR_REDIRECT_URI>&client_id=<YOUR_CLIENT_ID>&client_secret=<YOUR_CLIENT_SECRET>'
+     ```
+
+   This command will return the access token necessary for API calls.
+
+   ```json
+   {
+     "token_type": "bearer",
+     "refresh_token": "<Refresh Token>",
+     "access_token": "<Access Token>",
+     "expires_in": 1800
+   }
+   ```
+
+5. Store the access token securely for use in your application.
 
 ## Quickstart
 
-[//]: # (TODO: Add a quickstart guide to demonstrate a basic functionality of the module, including sample code snippets.)
+[//]: # "TODO: Add a quickstart guide to demonstrate a basic functionality of the module, including sample code snippets."
+
+To use the `HubSpot Marketing Events` connector in your Ballerina application, update the `.bal` file as follows:
+
+### Step 1: Import the module
+
+Import the `hubspot.marketing.events` module.
+
+```
+import ballerinax/hubspot.marketing.events as hsmevents;
+```
+
+### Step 2: Instantiate a new connector
+
+1. Create a `config.toml` file and, configure the obtained credentials in the above steps as follows:
+
+   ```
+   token = "<Access Token>"
+   ```
+
+   <!-- TODO Update -->
+
+2. Create a `hsmevents:ConnectionConfig` with the obtained access token and initialize the connector with it.
+
+<!-- TODO Add Screenshot -->
+
+### Step 3: Use Connector Operations
+
+Now, utilize the available connector operations.
```

Comment:
Can you update these sections as well ? Please refer to the Ballerina Zendesk connector [1]

[1] - https://github.com/ballerina-platform/module-ballerinax-zendesk/blob/main/ballerina/Package.md#quickstart